components/gnome/mousetweaks/Makefile

```diff
@@ -0,0 +1,65 @@
+#
+# CDDL HEADER START
+#
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License (the "License").
+# You may not use this file except in compliance with the License.
+#
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or http://www.opensolaris.org/os/licensing.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+# When distributing Covered Code, include this CDDL HEADER in each
+# file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+# If applicable, add the following below this CDDL HEADER, with the
+# fields enclosed by brackets "[]" replaced with your own identifying
+# information: Portions Copyright [yyyy] [name of copyright owner]
+#
+# CDDL HEADER END
+#
+
+#
+# Copyright (c) 2015, 2021, Oracle and/or its affiliates.
+#
+
+BUILD_BITS =            64
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		mousetweaks
+COMPONENT_VERSION=	3.32.0
+COMPONENT_PROJECT_URL=  https://wiki.gnome.org/Projects/Mousetweaks
+COMPONENT_ARCHIVE_HASH= \
+	sha256:0ebd5cdef01547787cbc3697ae758c57db1eec10eab613704d4a17b27685ae00
+COMPONENT_BUGDB=	gnome/accessibility
+COMPONENT_ANITYA_ID=	13153
+COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_BAID=		64951
+
+TEST_TARGET = $(NO_TESTS)
+
+include $(WS_MAKE_RULES)/common.mk
+include $(WS_MAKE_RULES)/gnome-component.mk
+
+# Assumes GNU xgettext in PATH
+CONFIGURE_ENV += PATH=$(GNUBIN):$(PATH)
+
+REQUIRED_PACKAGES += gnome/config/gsettings-desktop-schemas
+REQUIRED_PACKAGES += library/desktop/cairo
+REQUIRED_PACKAGES += library/desktop/gtk3
+REQUIRED_PACKAGES += library/desktop/pango
+REQUIRED_PACKAGES += library/glib2
+REQUIRED_PACKAGES += x11/library/libx11
+REQUIRED_PACKAGES += x11/library/libxcursor
+REQUIRED_PACKAGES += x11/library/libxfixes
+REQUIRED_PACKAGES += x11/library/libxtst
+# Auto-generated dependencies
+REQUIRED_PACKAGES += library/desktop/cairo
```

Comment:
I do not think it is good idea to bring this from `solaris-userland`.  We already diverged too much and many things makes no sense here when copied verbatim or are even counterproductive.

This is fairly simple component so package it from scratch should be straightforward.  And you would also get much smaller `Makefile`.  For example here are duplicate dependencies.  Above is `COMPONENT_BAID`, etc.